### PR TITLE
Js log files for once and always commands

### PR DESCRIPTION
**Pull request by jsingle**
@paetling

### CODE REVIEW

**Comment by paetling** [comment]
what does this really give the user.  Are we adding commands somewhere to formally expose these files.  Or are we going to document this somewhere so people know that these logs exist?

**Comment by jsingle** [comment]
I was thinking the latter.  Someone did ask for it at the install party

**Comment by jsingle** [comment]
Maybe the documentation change should also go in this PR

**Comment by paetling** [comment]
Yeah. Might also be worth a note on the change log

**Comment by jsingle** [comment]
Changelog status, dooooope

File: docs/changelog.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
   * `repo` and `mount` keys in app specs are now optional
 
+  * `always` and `once` scripts in app specs have their output logged to `/var/log/dusty_always.log` and `/var/log/dusty_once.log`, respectively. These are in-container locations
+
```
Comment (paetling) [inline docs/changelog.md]: NEW?

Comment (jsingle) [inline docs/changelog.md]: Are you saying it needs to start with "NEW: "?  What is the above bullet point doing?

Comment (thieman) [inline docs/changelog.md]: I think `NEW` should be for features or something that could enable new functionality. This is more a usability change.

**Comment by jsingle** [comment]
@paetling travis made a good point about "NEW"

**Pull request merged**